app/assets/javascripts/discourse/app/components/sidebar/back-to-forum.gjs

```diff
@@ -0,0 +1,28 @@
+import Component from "@glimmer/component";
+import { LinkTo } from "@ember/routing";
+import { service } from "@ember/service";
+import { defaultHomepage } from "discourse/lib/utilities";
+import dIcon from "discourse-common/helpers/d-icon";
+import i18n from "discourse-common/helpers/i18n";
+
+export default class BackToForum extends Component {
+  @service sidebarState;
+
+  get shouldDisplay() {
+    return this.sidebarState.currentPanel.filterable;
```

Comment:
IMO this should only display for certain panels, in this case if current panel is `ADMIN_PANEL`